readme.md

```diff
@@ -390,6 +390,7 @@ Thanks for contributing! 🦋🙌
 - [](# "prevent-pr-merge-panel-opening") Prevents the merge panel from automatically opening on every page load after it’s been opened once.
 - [](# "command-palette-navigation-shortcuts") Adds keyboard shortcuts to select items in command palette using <kbd>ctrl</kbd> <kbd>n</kbd> and <kbd>ctrl</kbd> <kbd>p</kbd> (macOS only).
 - [](# "submission-via-ctrl-enter-everywhere") Enables submission via <kbd>ctrl</kbd> <kbd>enter</kbd> on every page possible.
+- [](# "editor-link-new-tab") [Opens links in rich text editor in new tab instead of navigating away from the page.](https://github.com/refined-github/refined-github/assets/17681399/e18ca488-e055-4e41-943c-6243d9582bb7)
```

Comment:
This is a perfect demo, thank you!

However:

- Only pngs and gifs are allowed
- This type of link requires a redirect, so we should use the final address instead, which includes the file extension.